EdgeJenkinsfile

```diff
@@ -1,198 +1,4 @@
-import hudson.Util;
-import net.sf.json.JSONArray;
-import net.sf.json.JSONObject;
+#!/usr/bin/env groovy
+@Library('edge-ci@devicedb_ci') _
```

Comment:
please use master branch before merging

Reply:
yea changed the branch to master in the following commit.